### DISCUSSION TITLE
Deploy ASR models on triton

### DISCUSSION THREAD

**Discussion by hamjam** [NVIDIA-NeMo/NeMo#10327] [General]
Hello,
I want to deploy my fine-tuned FastConformer model on nvidia triton for batch inference and other functionalities. 
Are there any guide or script to do ASR model deployment on triton server?
I know there are some scripts in [nemo/deploy]( https://github.com/NVIDIA/NeMo/tree/main/nemo/deploy), but I believe these scripts are for llms implemented in NeMo.

**Comment by jayakommuru** [comment]
@hamjam were you able to find an alternate for this ?
